src/Contrib/Zipkin/SpanConverter.php

```diff
@@ -131,4 +145,127 @@ private static function toSpanKind(SpanDataInterface $span): ?int
         
         return null;
     }
+
+    private static function toAnnotation(EventInterface $event): array
+    {
+        $eventName = $event->getName();
+        $attributesAsJson = SpanConverter::convertEventAttributesToJson($event);
```

Comment:
If you would check here for ``count($event->getAttributes()) === 0``, you could simplify it a bit.

Reply:
I feel like I should get what you mean by this, but am struggling to at the moment. Is your idea to move the check on line 168 into this method, maybe using it in the ternary instead? 

I'm not sure how to then handle the json_encode failure case, but maybe that's a separate thing

Reply:
Yeah, something like that, but it's honestly not that important. :)